scripts/sync_ros_packages.py

```diff
@@ -90,6 +90,37 @@
 
         updates_generator.add_update_element(conf.UpdateElement(**d))
 
+        package_architecture = 'source' if arch == 'source' else 'binary-' + arch
+        # Compute the expected diff for this update element.
+        target_url = diff_repos.construct_packages_url(
+            target_repo_url,
+            ubuntu_distro,
+            'main',
+            package_architecture)
+        upstream_url = diff_repos.construct_packages_url(
+            upstream_repo_url,
+            ubuntu_distro,
+            'main',
+            package_architecture)
+        try:
+            pf_old = diff_repos.get_packagefile_from_url(target_url, name='old')
+        except RuntimeError, ex:
+            print("Exeception: %s \n NOT Computing diff" % ex, file=sys.stderr)
```

Comment:
Are these exceptions not fatal? They will for sure not be noted if the job is still "green".